cg/cli/workflow/mutant/base.py

```diff
@@ -99,3 +99,39 @@ def start_available(context: click.Context, dry_run: bool = False):
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+@mutant.command("store-available")
+@OPTION_DRY
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """Run QC checks and store bundles for all finished analyses in Housekeeper."""
+
+    analysis_api: MutantAnalysisAPI = context.obj.meta_apis["analysis_api"]
+
+    exit_code: int = EXIT_SUCCESS
+
+    analysis_api.run_qc_and_fail_analyses(dry_run=dry_run)
+
+    for case_obj in analysis_api.get_cases_to_store():
+        LOG.info("Storing deliverables for %s", case_obj.internal_id)
+        try:
+            context.invoke(store, case_id=case_obj.internal_id, dry_run=dry_run)
+        except Exception as exception_object:
+            LOG.error("Error storing %s: %s", case_obj.internal_id, exception_object)
+            exit_code = EXIT_FAIL
+    if exit_code:
+        raise click.Abort
+
+
+@mutant.command("run-qc")
+@click.pass_context
+def run_qc(context: click.Context, case_id: str) -> None:
+    """
+    Run QC on case and generate QC_report file.
+    """
+    analysis_api: MutantAnalysisAPI = context.obj.meta_apis["analysis_api"]
+
+    case: Case = analysis_api.status_db.get_case_by_internal_id(case_id)
```

Comment:
CLI commands should be as thin as possible, consider moving this line into the `run_qc` function and only pass the case id. Ideally, a CLI command only contains a single line of code.

Reply:
Done!